common/src/main/java/com/hammy275/immersivemc/client/config/screen/ItemGuideCustomizeScreen.java

```diff
@@ -136,13 +136,39 @@ protected void init() {
                         (newMode) -> {
                             ImmersiveMCConfig.placementGuideMode.set(newMode.ordinal());
                             ImmersiveMCConfig.placementGuideMode.save();
-                            ActiveConfig.loadConfigFromFile();
+
+                            //we don't use loadConfigFromFile here so that other guide values aren't accidentally overwritten
+                            ActiveConfig.placementGuideMode = newMode;
                         }
                 )
         );
 
+
         for (int i = 0; i < types.length; i++) {
             RGBA color = i == 0 ? ActiveConfig.itemGuideColor : i == 1 ? ActiveConfig.itemGuideSelectedColor : ActiveConfig.rangedGrabColor;
+
+            if(i == 0)
+            {
+                String sizeKey = "config." + ImmersiveMC.MOD_ID + "." + types[i] + "_size";
+                this.list.addBig(ScreenUtils.createIntSlider(
+                                sizeKey, (value) -> Component.literal(I18n.get(sizeKey) + ": " + String.format("%.02f", (float) value / 100.0f)),
+                                0, 100, (int)(ActiveConfig.itemGuideSize * 100),
+                                 (newVal) -> ActiveConfig.itemGuideSize = (float) newVal / 100.0f
+                        )
+                );
+            }
+            else if(i == 1)
+            {
```

Comment:
nitpick: Style with the rest of the codebase

```suggestion
            } else if(i == 1) {
```